### PR TITLE
Be able to verify bot-cfg.yaml locally inside a container

### PR DESCRIPTION
**Pull request by rpitonak**
One possible solution.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -30,6 +30,10 @@ test-in-container: test-build
 image-build:
 	docker build --tag ${IMAGE_NAME} .
 
+validate-bot-cfg: image-build
+	docker run -it -v ${BOT_CFG_PATH}:/tmp/bot-cfg.yml:ro \
+	${IMAGE_NAME} bash -c "DEPLOYMENT=prod python3 /tmp/frambo/frambo/config.py /tmp/bot-cfg.yml"
```
Comment (jpopelka) [inline Makefile]: `prod`? Could it be `dev`?

File: Makefile [hunk 2]
```diff
@@ -30,6 +30,10 @@ test-in-container: test-build
 image-build:
 	docker build --tag ${IMAGE_NAME} .
 
+validate-bot-cfg: image-build
+	docker run -it -v ${BOT_CFG_PATH}:/tmp/bot-cfg.yml:ro \
```
Comment (jpopelka) [inline Makefile]: `:ro` -> `:ro,Z`

File: README.md [hunk 3]
```diff
@@ -82,3 +82,13 @@ config parser will select item whose 'deployment' matches DEPLOYMENT environment
...
 If you have any questions don't hesitate to ask and we'll update the documentation with answers.
+
+### Validation of `bot-cfg.yml`
+
+You can easily verify locally that `bot-cfg.yaml` for your repository is valid.
+
+```
+$ git clone https://github.com/user-cont/frambo
+$ cd frambo
+$ make validate-bot-cfg BOT_CFG_PATH=<PATH>
```
Comment (jpopelka) [inline README.md:93]: I'm used to see `VARIABLE=value command arg` syntax. (I'm wondering that your's works too :-)  )

Comment (TomasTomecek) [reply README.md:93]: `make target VAR=content` is a make thing while yours is a shell thing; depends how you define the variables in the makefile, some of them might not be picked up with `VAR=content make target`, hence the former one is preferred afaik

**Review by jpopelka — APPROVED** [review]

**Pull request merged**